cli/rudder-cli

```diff
@@ -32,9 +34,10 @@ import requests
 import json
 import os
 import re
-import sys
 
-requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)
+import urllib3
+urllib3.disable_warnings()
```

Comment:
You have to do both in a try catch because request can use either one depending on situaion

Reply:
we do only the later in ncf, i figured it would be safe

Reply:
that's why we still have many unsecure request warnings log on some servers